betterlockscreen

```diff
@@ -358,14 +359,14 @@ get_wall_list() {
     elif [ "${#paths[@]}" -eq 1 ]; then
         for ((i=0; i<${#DISPLAY_LIST[@]}; i++)); do
```

Comment:
Why are we iterating a list known have only 1 item?

Reply:
I like to focus only on things I need to change, so I didn't modify code I didn't need to. Anyway I can add this to PR.

Reply:
Oops, sorry this wasn't yours, I just had a little target fixation. Don't worry about it.